src/common/com/intellij/plugins/haxe/ide/HaxeCompilerCompletionContributor.java

```diff
@@ -120,16 +123,24 @@ protected void addCompletions(@NotNull CompletionParameters parameters,
                        //Export/flash/haxe contains build.hxml which gets generated after build
                        break;
                      case HaxeModuleSettingsBaseImpl.USE_OPENFL:
-                       commandLineArguments.add(HaxelibCommandUtils.getHaxelibPath(moduleForFile));
-                       commandLineArguments.add("run");
-                       commandLineArguments.add("lime");
-                       commandLineArguments.add("display");
-                       //flash, html5, linux, etc
                        String targetFlag = moduleSettings.getNmeTarget().getTargetFlag();
```

Comment:
Another things I noticed when I worked on this issue, is that perhaps target should be `moduleSettings.getOpenFLTarget()`